### PR TITLE
Multi leaflet layer

### LINKED ISSUE #9: Display multiple GeoJSON in React-LeafletJS

### LINKED ISSUE #28: Colorize Leaflet Map

### LINKED ISSUE #29: Find Color Scheme

### PR DESCRIPTION
**Pull request by Mozartuss**
FInish Map

### CODE REVIEW

File: src/components/InteractiveMap.tsx [hunk 1]
```diff
@@ -1,66 +1,189 @@
...
+            let minValue = Infinity;
+            let maxValue = -Infinity;
+            Object.values(getDataRB)
+                .flat(1)
+                .forEach((dataEntry) => {
+                    const value = dataEntry.used_area_percent;
+                    minValue = minValue > value ? value : minValue;
+                    maxValue = maxValue < value ? value : maxValue;
+                });
+            return [0, maxValue];
```
Comment (fschmalzel) [inline src/components/InteractiveMap.tsx:38]: Why calculate minValue if it's not used?

File: src/components/InteractiveMap.tsx [hunk 2]
```diff
@@ -1,66 +1,189 @@
...
+            let minValue = Infinity;
+            let maxValue = -Infinity;
+            Object.values(getDataLK)
+                .flat(1)
+                .forEach((dataEntry) => {
+                    const value = dataEntry.used_area_percent;
+                    minValue = minValue > value ? value : minValue;
+                    maxValue = maxValue < value ? value : maxValue;
+                });
+            return [0, maxValue];
```
Comment (fschmalzel) [inline src/components/InteractiveMap.tsx:51]: Same here

File: src/components/InteractiveMap.tsx (hunk 1 above)
Comment (fschmalzel) [inline src/components/InteractiveMap.tsx:38]: Why calculate minValue if it's not used?

**Review by fschmalzel — APPROVED** [review]
lgtm

Altough i haven't really looked too deep into it. (No experience with Leaflet and GeoJSON)

The minValues can stay for now, as long as we remove them from the final draft.

File: src/components/InteractiveMap.tsx (hunk 2 above)
Comment (Mozartuss) [reply src/components/InteractiveMap.tsx:51]: I didn't know how we calculate the relative distribution, two approaches:
1. MinValue → MaxValue: Results in hard distribution of the individual values  → no homogeneous color gradient
2. 0% → MaxValue: homogeneous color gradient

That's the reason I left the MinValue calculation to switch back if one approach isn't fit our use case.

**Pull request merged**